lib/get-config.js

```diff
@@ -19,7 +22,11 @@ const ALLOWED_ERROR_CODES = [
 ];
 
 function resolveConfigPath(configPath, pluginPath) {
-  let usedPath = path.resolve(process.cwd(), configPath || '.template-lintrc');
+  let usedPath = path.resolve(process.cwd(), configPath || CONFIG_FILE_NAME);
```

Comment:
I think this might be slightly wrong, if we were provided a config path (e.g. from the bin script) we must use it **exactly** (no find-up). The only time find-up should be used is if we do not have a specified config path.

Reply:
fixed, but for me it looks like an "feature" because config path may be just file name, and we able to lookup "upper" non-standard file names

Reply:
Config path must match a specific on disk file (when not absolute, then it would be relative to CWD). If we want another "config file name" thing, we can add it but that is a separate thing (and I'm not sure that we should 😸 ).